### PR TITLE
Various fixes (for #25, #39 and #26)

### PR DESCRIPTION
**Pull request by mounirlamouri**

### CODE REVIEW

**Comment by mounirlamouri** [comment]
@marcoscaceres, can you have a look?

File: index.html [hunk 1]
```diff
@@ -570,34 +573,31 @@
...
           [[!PAGE-VISIBILITY]] but only one of those <a>Document</a>s is
-          focused, the <a>active orientation lock</a> MUST be the orientation
-          lock that applies to that <a>Document</a>'s <a>browsing context</a>.
+          focused, the <a>active orientation lock</a> is the orientation lock
+          that applies to that <a>Document</a>'s <a>browsing context</a>.
           </li>
           <li>Otherwise, the <a>active orientation lock</a> SHOULD be the
           orientation lock of the latest focused <a>Document</a>'s <a>top-level
-          browsing context</a>.
+          browsing context</a>, unless stated otherwise by the platform
```
Comment (marcoscaceres) [inline index.html:599]: nit: maybe instead of "unless stated otherwise" you could say "unless imposed otherwise"?

**Comment by marcoscaceres** [comment]
LGTM. It's ok if you want to ignore the nit.

**Comment by mounirlamouri** [comment]
I wouldn't use "impose" for platform conventions. Conventions can be ignored if you have a good reason to.

**Pull request merged**